config/jacoco.gradle

```diff
@@ -1,14 +1,21 @@
 apply plugin: 'jacoco'
 
 jacoco {
-    toolVersion = "0.8.3"
+    toolVersion = jacoco_core_version
     reportsDir = file("$buildDir/reports/jacoco")
 }
 
-task jacocoTestReport(type: JacocoReport, dependsOn: "testStagingDebugUnitTest") {
+tasks.withType(Test) {
+    jacoco.includeNoLocationClasses = true
+}
+
+task jacocoTestReport(type: JacocoReport) {
```

Comment:
we already had this task, so I think that's why you won't need it, or, check again if u can use it and remove this task entirely

Reply:
The reason why you're suggesting to remove this, is because it's supposed to be integrated in that plugin already, right?

Reply:
yes if the plugin work per our expectation then we can make the template leaner by using just the plugin config instead of writing gradle Task ourselves